api/src/main/resources/application.properties

```diff
@@ -9,14 +9,14 @@ spring.mvc.log-request-details=${SPRING_SHOW_REQUEST_DETAILS}
 spring.datasource.url=${JDBC_URL}
 spring.datasource.username=${ORACLE_USERNAME}
 spring.datasource.password=${ORACLE_PASSWORD}
-spring.jpa.database-platform=org.hibernate.dialect.Oracle12cDialect
+spring.jpa.database-platform=org.hibernate.dialect.OracleDialect
```

Comment:
this was done to remove warning about deprecated OracleDialect setting.

[org.hibernate.orm.deprecation] | [HHH90000026: Oracle12cDialect has been deprecated; use org.hibernate.dialect.OracleDialect instead ] |